File: pset1/moremario/mario.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    int n = 0;

    do
    {
        printf("Full pyramid’s height: ");
        n=get_int();
        if(n < 0 || n > 23)
        {
            printf("Full pyramid’s height is a non-negative integer no greater than 23. Retry. ");
        }
    }
    while(n < 0 || n > 23);

    for(int r=0; r<n; r++)
    {
        for(int space=(n-r)-1; space>0; space--)
        {
            printf(" ");
        }

        for(int hash=0; hash<r+1; hash++)
        {
            printf("#");
        }

        printf("  ");

        for(int hash=0; hash<r+1; hash++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario.py
#define height as variable n
n=int(input('Enter the height of the half-pyramid Mario is climbing: '))

#prompt user if n>23 or n<1
while n>23 or n<1:
    print('Height of half-pyramid must be a positive non-zero integer, no greater than 23. Try Again.')
    n=int(input('Enter the height of the half-pyramid Mario is climbing: '))

#define space and hash as variables s and h respectively
s=n-1
h=2
while s>=0:
    print(' '*s + '#'*h)
    s-=1
    h+=1